docker/sandbox-bundled/Makefile

```diff
@@ -38,12 +47,7 @@ manifests: dep_update
 		kustomize/complete-agent > manifests/complete-agent.yaml
 
 .PHONY: build
-build: flyte dep_update manifests
-	[ -n "$(shell docker buildx ls | awk '/^flyte-sandbox / {print $$1}')" ] || \
-		docker buildx create --name flyte-sandbox \
-		--driver docker-container --driver-opt image=moby/buildkit:master \
-		--buildkitd-flags '--allow-insecure-entitlement security.insecure' \
-		--platform linux/arm64,linux/amd64
+build: create_builder flyte dep_update manifests
```

Comment:
Need some tiny change, cause `create_builder` will be called in `flyte`

Reply:
Should we remove repetitive called?

Reply:
done